<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bgwidget</class>
 <widget class="QDialog" name="bgwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget

#bgwidget { background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(92, 149, 255, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 48pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Diabete Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_btn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>510</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(233, 185, 110);
font: 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Signup</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailfield">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordfield">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 41, 41);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(252, 233, 79);
font: 32pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
